tests/providers/test_phone_number.py

```diff
@@ -285,6 +285,21 @@ def test_phone_number(self, faker, num_samples):
             assert pattern.fullmatch(phone_number)
 
 
+class TestEsPE:
+    """Test es_PE phone number provider methods"""
+
+    def test_phone_number(self, faker, num_samples):
+        pattern: Pattern = re.compile(
+            r"(\+?51|\(\+51\))?(9\d{8})|"
+            r"(\+?51|\(\+51\))?((9\d{2}) \d{3} \d{3})|"
+            r"(9\d{8})|"
+            r"((9\d{2}) \d{3} \d{3})"
+        )
+        for _ in range(num_samples):
+            phone_number = faker.phone_number_pe()
```

Comment:
the method is called `phone_number()`:

```suggestion
            phone_number = faker.phone_number()
```